Makefile

```diff
@@ -1,3 +1,13 @@
+TESTS_TO_RUN := $(shell go list ./... | grep -v integrationtests | grep -v mock)
+
+
 test:
 	@echo "  >  Running unit tests"
-	go test -cover -race -coverprofile=coverage.txt -covermode=atomic -v ./...
+	go test -cover -race -coverprofile=coverage.txt -covermode=atomic -v ${TESTS_TO_RUN}
+
+integration-tests:
+	@echo " > Running integration tests"
+	cd scripts && ./script.sh start
+	go test -v ./integrationtests
+	cd scripts && ./script.sh delete
+	cd scripts && ./script.sh stop
```

Comment:
`delete` will remove all indexed  data from Elasticsearch cluster
`stop`-  will turn off the docker container. 
Both steps are optional because ofter the actions run cleanup is done automatically by Github

Reply:
ok